torch/nn/modules/container.py

```diff
@@ -15,6 +15,19 @@
 T = TypeVar('T', bound=Module)
 
 
+# Copied from torch.nn.modules.module, required for a cusom __repr__ for ModuleList
```

Comment:
I don't have a strong preference here. I made a copy because it's a short function and because this way it is independent of the version in `torch.nn.modules.module`. Might be useful in case any future changes will be incompatible between regular modules and `ModuleList`.

Reply:
Sounds good!